content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,169 @@
+---
+title: >
+  Turn Visual Studio Code into an ML experimentation platform with the DVC
+  extension
+date: 2022-06-14
+description: >
+  Today we are releasing the DVC extension, which brings a full ML
+  experimentation platform to VS Code.
+descriptionLong: >
+  Today we are releasing the DVC extension, which brings a full ML
+  experimentation platform to VS Code. Manage your data, run experiments,
+  compare metrics, and visualize plots, all from the comfort of your IDE.
+picture: 2022-06-14/header.png
+author: rob_dewit
+# commentsUrl:
+tags:
+  - VS Code
+  - DVC
+  - MLOps
+  - Release
+---
+
+Since its beta release in 2017, DVC has become an essential tool for many data
+science teams. Its data versioning capabilities, reproducible pipelines, and
+experiment tracking features are at the core of our ecosystem of open MLOps
+tools.
+
+Today we are proud to launch a new product that extends how ML teams can use
+DVC: our extension for Visual Studio Code.
+
+With this extension you get a full VS Code-native experimentation platform for
+your machine learning projects. You can control datasets and models, run
+experiments, view metrics, create plots, and much more. All in one place in your
+IDE.
+
+[Install the DVC extension from the marketplace to get
+started.](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+
+# Why a VS Code extension?
+
+We built DVC to expand upon the Git workflow to make it well-suited for ML
+experimentation. This approach brought us independence from the infrastructure and
+provided a natural connection to best practices from software engineering. 
+However, a pure CLI tool can only take things so far when
+it comes to visualizing experiments or displaying large tables.
+
+[VS Code is the IDE of choice for
+many](https://insights.stackoverflow.com/survey/2021#section-most-popular-technologies-integrated-development-environment)
+and was a natural choice for a platform to add a graphical interface to DVC.
+
+With this extension, we want to:
+
+- Move the ML experimentation workflow into your IDE
+- Provide interactive plots and tables for analyzing ML experiments
+- Make DVC more accessible by providing an alternative to the complexity of the
+  CLI
+
+As data scientists, DVC is our toolbox. This extension turns VS Code into our
+workshop.
+
+# Features
+
+Here are the main features that are available when you install the extension
```

Comment:
This is awkward and unnecessary.  Just change "Features" to "Main Features". Then when you get to What's next it makes more sense too.

Reply:
Reworked in new version. Less awkward now?